src/dmd/parse.d

```diff
@@ -6162,7 +6160,7 @@ final class Parser(AST) : Lexer
                     {
                         t = peek(t);
                         if (t.value != TOKsemicolon && t.value != TOKcomma && t.value != TOKrbracket && t.value != TOKrcurly)
-                            goto Lexpression;
+                            goto default;
```

Comment:
The CI error is probably due to the `default` being at L6171.

Reply:
Ah, I had a nagging doubt that this one might be too ambitious.